src/bikeracks/schema.sql

```diff
@@ -76,6 +77,14 @@ CREATE TABLE suggested_removals (
     reason INTEGER
 )
 
+/* Maps reason integer from suggested_removals table to a language
+and reason */
+CREATE TABLE removal_reasons (
+    reason_id INTEGER PRIMARY KEY AUTOINCREMENT,
```

Comment:
Can you put a foreign key constraint between `bikeracks.reason` and `removal_reasons.reason_id` ?

Also name them the same way `removal_reason_id` or something.

Reply:
Done.

Reply:
Since you're always going to manually insert these, `AUTOINCREMENT` doesn't really make a difference. I'd remove it so that it's clear that an id must be chosen explicitly for each reason and stay the same forever.

Also I think you want most of these columns to be `NOT NULL`?